rclcpp_action/include/rclcpp_action/server_goal_handle.hpp

```diff
@@ -196,7 +196,7 @@ class ServerGoalHandle : public ServerGoalHandleBase
 
   /// Indicate that a goal has been canceled.
   /**
-   * Only call this if the goal is executing or pending, but has been canceled.
+   * Only call this if the goal is canceling, but has been canceled.
```

Comment:
I'm not 100% sure of this, but I think this will be clearer like this:

```suggestion
   * Only call this if the goal is canceling.
```